pkg/router/template/configmanager/haproxy/manager.go

```diff
@@ -1105,6 +1111,16 @@ func modAnnotationsList(termination routev1.TLSTerminationType) []string {
 		"router.openshift.io/haproxy.health.check.interval",
 	}
 
+	// allow adding list of extra annotations for HAProxy blueprint
+	listExtraAnnotations, ok := os.LookupEnv("EXTRA_ANNOTATIONS_FOR_BLUEPRINT")
```

Comment:
We don't really need to check if the environment variable is defined do we? 
We could just use `os.Getenv` and proceed on. One less check ... 

Or if you want to use `cmdutil` that's an alternative ala:  
```envValue := cmd.Env("ROUTER_BLUEPRINT_CUSTOM_ANNOTATIONS", "")```
(needs to import ```cmdutil "github.com/openshift/origin/pkg/cmd/util"```).

Am ok either way.

But change the environment variable name to something like `ROUTER_BLUEPRINT_CUSTOM_ANNOTATIONS`.